public/app/config/env/staging.js

```diff
@@ -0,0 +1,22 @@
+(function () {
+    'use strict';
+
+    define(
+        [
+            'util/container',
+            'services/searchService'
+        ],
+        function (container, searchService) {
+            var simpleApiBaseUrl = 'https://staging-collections.histwest.org.au/service.php/simple',
```

Comment:
Sorry I meant manual testing, not automated testing, and specifically hitting the staging instance, would its CORS configuration allow being hit from other machines, etc.  I'm happy with just deploying it into staging and trying it then, and if there's any issues we can just fix it.